frontend/libs/studio-feedback-form/src/config/FeedbackFormImpl.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import type { ButtonTexts, QuestionConfig } from '../types/QuestionsProps';
+import { FeedbackFormContextProvider } from '../contexts/FeedbackFormContext';
+import { FeedbackForm } from '../FeedbackForm/FeedbackForm';
+
+export class FeedbackFormImpl {
+  private readonly buttonTexts: ButtonTexts;
+  private readonly heading: string;
+  private readonly questions: QuestionConfig[];
+  private readonly position: 'inline' | 'fixed' = 'inline';
+  private readonly onSubmit: (answers: Record<string, any>) => void;
+
+  constructor(config: {
+    buttonTexts: ButtonTexts;
+    heading: string;
+    questions: QuestionConfig[];
+    position?: 'inline' | 'fixed';
+    onSubmit: (answers: Record<string, any>) => void;
```

Comment:
nit: ref. antother comment, I think we can do the same here to get rid of any? 🤔 
```suggestion
    onSubmit: (answers: Record<string, Array<string>) => void;
```